zone/bot.cpp

```diff
@@ -625,19 +634,7 @@ NPCType *Bot::FillNPCTypeStruct(
 	n->bodytype = 1;
 	n->deity = EQ::deity::DeityAgnostic;
 	n->level = botLevel;
-	//n->npc_id = 0;
```

Comment:
Why are all of these fills removed

Reply:
I eventually want to move Bots out of the NPC class, and just be a derived class of Mob. 

Having ```Mob() -> NPC() -> Bot()``` is needlessly complex, and Bots don't actually use much NPC logic via polymorphism, instead they have their own logic.

So I just figured I'd remove these fill comments. but guess I should have waited until I did my actual full cleanup.